libkmod/libkmod-elf.c

```diff
@@ -470,13 +470,16 @@ int kmod_elf_get_strings(const struct kmod_elf *elf, const char *section, char *
 	if (strings[i - 1] != '\0')
 		count++;
 
+	/* (string vector + NULL) * sizeof(char *) + size + extra NULL */
```

Comment:
the last "extra NULL" should be NUL to highlight that we are talking about '\0' not (void *)0.

Reply:
NUL is a single byte. NULL is a pointer, i.e. 4 or 8 bytes.